packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -867,8 +908,18 @@ export class FeatureDevController {
         // Re-run the opening flow, where we check auth + create a session
         await this.tabOpened(message)
 
-        this.messenger.sendChatInputEnabled(message.tabID, true)
-        this.messenger.sendUpdatePlaceholder(message.tabID, i18n('AWS.amazonq.featureDev.placeholder.describe'))
+        if (prefilledPrompt) {
+            this.messenger.sendAnswer({
```

Comment:
Non-blocking, but why new task has to couple with dev file generation?

```
this.messenger.sendAnswer({
                type: 'system-prompt',
                tabID: message.tabID,
                message: i18n('AWS.amazonq.featureDev.pillText.generateDevFile'),
            })
this.newTask(data, generateDevFilePrompt)
```

Reply:
good catch, let me change that